### PR TITLE
Remove formplayer backend

### PR DESCRIPTION
**Pull request by wpride**
cross-request: https://github.com/dimagi/touchforms/pull/276

### CODE REVIEW

**Comment by czue** [comment]
whoa does this mean we're fully converting everyone over?

**Comment by wpride** [comment]
@czue confirming that this won't move everyone onto Nimbus. This was an intermediary flag that allowed us to route calls from the old CloudCare frontend to the Touchforms server to Formplayer instead so that we could validate Formplayer was behaving correctly. Only a few test domains on here.

**Comment by czue** [comment]
got it thanks

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js [hunk 1]
```diff
@@ -151,36 +150,18 @@ WebFormSession.prototype.serverRequest = function (requestParams, callback, bloc
     this.numPendingRequests++;
     this.onLoading();
 
-    if (self.formplayerEnabled){
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js]: this we need

Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js]: https://github.com/dimagi/commcare-hq/blob/br/resize-debugger/corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js#L133

**Review by benrudolph — CHANGES_REQUESTED** [review]

**Pull request closed without merging**